File: javaToC++/ex3.java
/*
Class: CPSC 122
Section: 01
Team Member 1: Paul De Palma
Team Member 2: None
Submitted By: Paul De Palma
GU Username: depalma
File Name: CForLoop.java
Program illustrates a for loop 
To Build: javac ex3.java
To Execute: java ex3 
*/

import java.util.Scanner;

public class ex3 
{
  public static void main(String[] args)
  {
    int howMany;

    Scanner keyboard = new Scanner(System.in);


    System.out.println("How many integers do you want to see?");
    howMany = keyboard.nextInt();

    for (int i = 0; i < howMany; i++)
      System.out.println(i);

  }
} 

File: A-JavaToC++/ex3.cpp




/*
Name: Paul De Palma
Class: CPSC122, Section 1
Date: January 17, 2021
Assignment: Example 3 
Description: Program illustrates console i/o and a for loop;
*/

#include <iostream>
using namespace std;


int main()
{
  int howMany; 

  cout << "How many integers do you want to see?" << endl; 
  cin >> howMany;

  for (int i = 0; i < howMany; i++)
    cout << i << endl;

  return 0;
} 


